cpp/src/io/parquet/page_data.cu

```diff
@@ -1200,11 +1200,11 @@ static __device__ void gpuUpdateValidityOffsetsAndRowIndices(int32_t target_inpu
                                                              int t)
 {
   // max nesting depth of the column
-  int max_depth = s->col.max_level[level_type::REPETITION];
+  short max_depth = s->col.max_level[level_type::REPETITION];
   // how many (input) values we've processed in the page so far
   int input_value_count = s->input_value_count;
   // how many rows we've processed in the page so far
-  int input_row_count = s->input_row_count;
+  short input_row_count = s->input_row_count;
```

Comment:
Does this shrink memory consumption?
For resource usage, after compilation 
`cuobjdump  --dump-resource-usage ./CMakeFiles/cudf.dir/src/io/parquet/page_data.cu.o | c++filt`
Compare.